Views/CarouselItem-ResponsiveMediaField.liquid

```diff
@@ -0,0 +1,11 @@
+﻿{% if Model.HasMedia %}
+    {% for item in Model.Media %}
+        <picture class="card__background">

```

Comment:
Don't think this should have card__background as a default class... It should either have nothing, or something like `image image--responsive`

Reply:
Oops, I misread the filename. Resolving.